docs/setup/keadm.md

```diff
@@ -96,70 +96,308 @@ KubeEdge edgecore is running, For logs visit:  /var/log/kubeedge/edgecore.log
 ```
 
 ### Enable `kubectl logs` Feature
+#### 1. Environments:
```

Comment:
No need to write a specific environment.

Reply:
Okay, I will remove it.